ambillion-ui/src/components/common/breadCrumb/breadCrumb.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+
+type BreadcrumbProps = {
+    title: string;
+};
+
+export const Breadcrumb: React.FC<BreadcrumbProps> = ({ title }) => {
```

Comment:
Breadcrumb component should be dynamic based on the URL routes. it should handle this kind of routes up to nth child routes. /products/addproducts.